src/components/PersonLink.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  return (
+    <Link to={`/people/${person.slug}`} className={classNames({ 'has-text-danger': person.sex === 'f' })}>
```

Comment:
Try not to use 'magic values', better will be to create a `const` value for this 'f'